TeachingRecordSystem/src/TeachingRecordSystem.Core/Jobs/EwcWalesImport/InductionImporter.cs

```diff
@@ -0,0 +1,452 @@
+                break;
+            case EwcWalesMatchStatus.TrnAndDateOfBirthMatchFailed:
+                errors.Add($"For TRN {row.ReferenceNumber} Date of Birth does not match with the existing record."); //No Test
+                break;
+            case EwcWalesMatchStatus.MultipleTrnMatched:
+                errors.Add($"TRN {row.ReferenceNumber} was matched to more than one record in the system.");
+                break;
+            case EwcWalesMatchStatus.TeacherInactive:
+                errors.Add($"Teacher with TRN {row.ReferenceNumber} is inactive."); //No Test
+                break;
+            case EwcWalesMatchStatus.TeacherHasQts:
+                errors.Add($"Teacher with TRN {row.ReferenceNumber} has QTS already.");
+                break;
+        }
+
+        if (!string.IsNullOrEmpty(row.EmployerCode))
+        {
+            switch (lookups.OrganisationMatchStatus)
+            {
+                case EwcWalesMatchStatus.NoMatch:
+                    validationFailures.Add($"Organisation with Induction Body Code {row.EmployerCode} was not found.");
+                    break;
+                case EwcWalesMatchStatus.MultipleMatchesFound:
+                    validationFailures.Add($"Multiple organisations with Induction Body Code {row.EmployerCode} found.");
+                    break;
+            }
+        }
+
+        //if teacher is exempt via set and doesn't have an induction
+        if (lookups.InductionMatchStatus == EwcWalesMatchStatus.NoMatch && lookups.Person != null && lookups.Person!.dfeta_qtlsdate.HasValue)
+        {
+            errors.Add("may need to update either/both the 'TRA induction status' and 'Overall induction status");
+        }
+
+        //if teacher is exempt via set and inductionstatus is not in permitted updatabe statuses
+        if (lookups.InductionMatchStatus == EwcWalesMatchStatus.OneMatch && lookups.Person != null && lookups.Person!.dfeta_qtlsdate.HasValue)
+        {
+            errors.Add("may need to update either/both the 'TRA induction status' and 'Overall induction status'");
+        }
+
+        if (lookups.Induction != null)
+        {
+            switch (lookups.Induction.GetAttributeValue<OptionSetValue>(dfeta_induction.Fields.dfeta_InductionStatus).Value)
+            {
+                case (int)dfeta_InductionStatus.Pass:
+                case (int)dfeta_InductionStatus.PassedinWales:
+                case (int)dfeta_InductionStatus.Exempt:
+                case (int)dfeta_InductionStatus.Fail:
+                case (int)dfeta_InductionStatus.FailedinWales:
+                    errors.Add($"Teacher with TRN {row.ReferenceNumber} completed induction already.");
+                    break;
+            }
+        }
+
+        return (validationFailures, errors);
+    }
+    public async Task<(EwcWalesMatchStatus, Guid? OrganisationId)> FindMatchingOrganisationsRecordAsync(string OrgNumber)
```

Comment:
I think these lookup methods should return their own enum rather than re-using `EwcWalesMatchStatus` e.g. `OrganizationLookupResult`, `InductionLookupResult`, `ContactLookupResult` etc.